lib/src/misc.ts

```diff
@@ -13,7 +13,7 @@ export default function (cv: typeof openCV): void {
      * non Natif code
      * @param type Mat type as int value
      */
-    cv.toMatTypeName = (type: number): MatTypes | undefined => {
+    cv.toMatTypeName = function toMatTypeName(type: number): MatTypes | undefined {
```

Comment:
Is the usage of `function` keywords change anything in this function ?
You may want to bind `cv` as this, but the `this` is not used.

Reply:
This shouldn't change anything other than giving the function a `prototype` and a `name`. None of the functions use `this`, so I don't think any `bind` is necessary.